develop.sh

```diff
@@ -3,6 +3,12 @@ set -euo pipefail
 IFS=$'\n\t'
 
 trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
-trunk serve --proxy-backend="http://[::1]:8081/api/" &
-cargo watch -- cargo run --bin server -- --port 8081 --ssl-port 8443 --static-dir client/dist --cert-dir cert
 
+# Export RUST_LOG for more detailed logging
+export RUST_LOG="debug,hyper=info,mio=info"
+
+# Start trunk with more verbose output
+trunk serve --verbose --proxy-backend="https://[::1]:8443/api/" &
```

Comment:
The most recent trunk version, 0.21.7-1 gives me an error `error: the argument '--verbose...' cannot be used with '--log <LOG>'`